simtools/utils/general.py

```diff
@@ -516,29 +477,6 @@ def set_default_kwargs(in_kwargs, **kwargs):
     return in_kwargs
 
 
-def sort_arrays(*args):
```

Comment:
I suggest to keep sort_arrays in general.py - this is quite generic.

Reply:
yet it is a transformation. You really meant geometrical transformations then